lib/manager/components/GeneralSettings.js

```diff
@@ -40,9 +40,11 @@ type State = {
 export default class GeneralSettings extends Component<Props, State> {
   state = {}
 
-  _onChange = ({target}: SyntheticInputEvent<HTMLInputElement>) => {
+  _onChange = ({target}: SyntheticInputEvent<HTMLInputElement>, key: 'name' | 'url') => {
```

Comment:
You don't need a `key` variable, you should be able to use `target.name` to pull the value from the html tag

Reply:
Astounding. Changed in https://github.com/ibi-group/datatools-ui/pull/841/commits/07ed8e52df2b04bd228f7bf2d850f03a41005f9a